model/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "model"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1"
+bdk = { version = "0.16", default-features = false }
+cached = { version = "0.30.0", default-features = false, features = ["proc_macro"] }
+chrono = { version = "0.4", features = ["serde"] }
+hex = "0.4"
+itertools = "0.10"
+maia = "0.1.0"
+nalgebra = { version = "0.30", default-features = false, features = ["std"] }
+ndarray = "0.15.4"
+ndarray_einsum_beta = "0.7.0"
+num = "0.4.0"
+parse-display = "0.5.3"
+reqwest = { version = "0.11", default-features = false }
```

Comment:
Why?

Reply:
I was unhappy about this too. We use the `Url` type. I wanted to depend on the `url` crate, but I wasn't sure if it would result in annoying incompatibilities between crates. What do you think?

Reply:
I think you can depend on `url`, `reqwest` just re-exports it in its latest version.